plover/main.py

```diff
@@ -48,14 +48,19 @@ def init_config_dir():
         os.makedirs(CONFIG_DIR)
 
     # Copy the default dictionary to the configuration directory.
-    if not os.path.exists(DEFAULT_DICTIONARY_FILE):
-        unified_dict = {}
-        dict_filenames = glob.glob(os.path.join(ASSETS_DIR, '*.json'))
-        for dict_filename in dict_filenames:
-            unified_dict.update(json.load(open(dict_filename, 'rb')))
-        ordered = OrderedDict(sorted(unified_dict.iteritems(), key=lambda x: x[1]))
-        outfile = open(DEFAULT_DICTIONARY_FILE, 'wb')
-        json.dump(ordered, outfile, indent=0, separators=(',', ': '))
+    def copy_dictionary_to_config(name):
+        source_path = os.path.join(ASSETS_DIR, name)
+        out_path = os.path.join(CONFIG_DIR, name)
+        if not os.path.exists(out_path):
```

Comment:
I'm okay with main.json, if you are. It's a real word, it's short and to the point. These dictionaries will only be made if the user doesn't have any dictionaries.

In that comment I was just talking about renaming the variable because "unified_dict" was the thing I replaced. A better piece would be something "unsorted_dict = json.load([...]" for lines 55 and 56